gsoc/gsoc2016/ideas_juliaastro.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title:  "Ideas for Julia Astro"
+show_main: false
+ideas_team: JuliaAstro
+---
+
+### Image compression and efficient table reading in FITSIO.jl
+
+*Suggested Mentor(s):* [Kyle Barbary](http://github.com/kbarbary)
```

Comment:
That might be tough... I seem to be the only developer interested in this at the moment! I'll ask a couple people personally if they'd be interested, but it might be that Julia Astro is still just too small at this point.